ansible-advancedlinux/azuredeploy.json

```diff
@@ -1,9 +1,11 @@
 {
+
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
     "location": {
       "type": "string",
+      "defaultValue": "West US",
```

Comment:
I've removed the default location, and added new locations in which we have Premium Storage. As this templates requires Premium Storage, I will keep a list of regions in which PS is available.